SevenZip/SevenZip.csproj

```diff
@@ -94,5 +94,6 @@
     <Reference Include="System.Configuration" Condition="'$(TargetFramework)' == 'net45'">
       <HintPath>C:\Windows\Microsoft.NET\assembly\GAC_MSIL\System.Configuration\v4.0_4.0.0.0__b03f5f7f11d50a3a\System.Configuration.dll</HintPath>
     </Reference>
+    <Reference Include="System.Configuration" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
I'm not sure if there should be `<HintPath>` here. Should it?

Reply:
I would actually merge the two instead:

`<Reference Include="System.Configuration" Condition="'$(TargetFramework)' == 'net45' OR '$(TargetFramework)' == 'net462'" />`

And yes, HintPath can be removed.

Reply:
merged and removed